research/picai/prepare_annotations.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import os
+
+import SimpleITK as sitk
+
+
+def prepare_annotations(human_annotations_dir: str, ai_annotations_dir: str, annotations_write_dir: str) -> None:
+    for filename in sorted(os.listdir(human_annotations_dir)):
+        path = os.path.join(human_annotations_dir, filename)
+        annotation_write_path = os.path.join(annotations_write_dir, filename)
+        if path.endswith(".nii.gz"):
+            annotation = sitk.ReadImage(path)
+            sitk.WriteImage(annotation, str(annotation_write_path), useCompression=True)
+
+    for filename in os.listdir(ai_annotations_dir):
+        path = os.path.join(ai_annotations_dir, filename)
+        annotation_write_path = os.path.join(annotations_write_dir, filename)
+        if path.endswith(".nii.gz") and not os.path.exists(annotation_write_path):
```

Comment:
Maybe a silly question, but why do we check if the path exists here but not for the human annotations. My guess is that there might be human annotations and ai annotations for the same file and we don't want to overwrite the human GT if it exists. A comment if that's true would be helpful

Reply:
Correct!